extensions/amp-consent/0.1/consent-info.js

```diff
@@ -63,6 +63,25 @@ export const CONSENT_ITEM_STATE = {
   // {unknown, accepted, rejected}
 };
 
+/**
+ * @enum {string}
+ * @visibleForTesting
+ */
+export const EXPOSED_CONSENT_API = {
+  TCF_POST_MESSAGE: 'tcfPostMessageApi',
+};
+
+/**
+ * @enum {string}
+ * @visibleForTesting
+ */
+export const TCF_POST_MESSAGE_API_COMMANDS = {
```

Comment:
Will we expand the list later on?

Reply:
We could in the future. Do you suggest we just change it to a string for now, and then if we do expand later on, make the change then?